src/components/app/Overview/Overview.tsx

```diff
@@ -280,25 +327,166 @@ export default function AppOverview({ appMetaInfo, getAppMetaInfoRes }: AppOverv
                     <RocketIcon className="icon-dim-20 scn-9 mr-8" />
                     Deployments
                 </div>
-                {otherEnvsLoading ? (
-                    <div className="dc__loading-dots" />
-                ) : renderDeploymentComponent()}
+                {otherEnvsLoading ? <div className="dc__loading-dots" /> : renderDeploymentComponent()}
+            </div>
+        )
+    }
+
+    const renderWorkflowsStatus = () => {
+        const renderWorkflowComponent = () => {
```

Comment:
Don't write functions that return JSX element inside other

Reply:
done